### DISCUSSION TITLE
Unavailability of the google drive directory because of access restrictions.

### DISCUSSION THREAD

**Discussion by Shashankss1205** [akdasUAF/ForestFireDetection#22] [Q&A] [answered]
Respected Mentor
Please make available the Google drive folder link so that I can setup the base code and data.

**Comment by YaliWang2019** [comment, marked as answer]
Hi Shashank,

I just updated that folder's link. It should be open to access now. Sorry for the inconvenience. 

Thanks,
Yali

**Reply by Shashankss1205** [reply]
Thankyou for your prompt response 😃
